smac/tae/base.py

```diff
@@ -247,18 +247,19 @@ def run_wrapper(
 
         # Catch NaN or inf.
         if (
-            self.run_obj == 'runtime' and not np.isfinite(runtime)
-            or self.run_obj == 'quality' and not np.all(np.isfinite(cost))
+            self.run_obj == "runtime"
+            and not np.isfinite(runtime)
+            or self.run_obj == "quality"
+            and not np.all(np.isfinite(cost))
```

Comment:
While the order of operations works here, I would bracket these statements together.
```python
if (
    (self.run_obj == "runtime" and not np.isfinite(runtime))
    or (self.run_obj == "quality" and not np.all(np.isfinite(cost)))
):

```